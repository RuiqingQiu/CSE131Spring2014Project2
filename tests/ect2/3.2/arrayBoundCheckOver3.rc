function : int main(){
   bool [10]b;
   int a = 155;
   b[a];

   return 0;
}
