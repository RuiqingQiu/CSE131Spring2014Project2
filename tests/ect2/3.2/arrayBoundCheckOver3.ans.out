Index value of 155 is outside legal range [0,10).
